HW1/P2/P2b.py

```diff
@@ -1,3 +1,31 @@
 from P2 import *
+import pyspark as py
+import matplotlib.pyplot as plt
+import random
 
-# Your code here
+def location(x):
+	return x/500.0 -2 ;
+
+#initiate spark
+#create sparkConf and a new Spark
+conf = py.SparkConf().setAppName("CS205HW1P2")
+sc = py.SparkContext();
+series = range(2000);
+random.shuffle(series);
```

Comment:
well done. I did not know this command. you could have used partitionBy and passed a map that randomly attributes the points to the partition.